recipes/sqlite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version="3.11.1" %}
+{% set version_split=version.split(".") %}
+{% set major=version_split[0] %}
+{% set minor=version_split[1]|int * 100 %}
+{% set bugfix=version_split[2]|int * 100 %}
+{% set version_coded=(major ~ (minor|string)[:3] ~ (bugfix|string)[:3]) %}
+
+package:
+  name: sqlite
+  version: {{ version }}
+
+source:
+  fn: sqlite-autoconf-{{ version_coded }}.tar.gz
+  url: https://www.sqlite.org/2016/sqlite-autoconf-{{ version_coded }}.tar.gz
+  sha1: c4b4dcd735a4daf5a2e2bb90f374484c8d4dad29
+
+build:
+  number: 0
+  features:
+    - vc9    # [win and py27]
+    - vc10   # [win and py34]
+    - vc14   # [win and py35]
+
+requirements:
+  build:
+    # python is for lining up vc feature (runtime)
+    - python      # [win]
+
+test:
+  requires:
+    - python  2.7*     # [win and py27]
```

Comment:
If changing it here doesn't fail the linter here, I'm fine with it. Though I'm also fine with leaving it alone. This is a pretty minor issue.